### ISSUE TITLE
Fix mtp connection with fx manager like bliss os

### ISSUE THREAD

**Issue by Arunkapil01** [bigdroid-hooks/darkmatter-hooks#1]
![Screenshot_20210703-002652](https://user-images.githubusercontent.com/25702790/124318067-ba787a80-db95-11eb-9ac2-601fd57d62f3.jpg)
Internal error when i try to connect my phone connected to pc via usb cable.

**Comment by MrMiy4mo** [comment]
MTP sucks for some devices, even with windows, i use an app to access files with local ftp server.

**Comment by Arunkapil01** [comment]
> MTP sucks for some devices, even with windows, i use an app to access files with local ftp server.

My phone can connect via usb cable and my files are shown in default android downloads file manager. But i can't Mount it in fx file manager like Bliss os. But i can't use Bliss because it doesn't support my drivers.

**Comment by Xtr126** [comment]
> MTP sucks for some devices, even with windows, i use an app to access files with local ftp server.

It does work very well in Bliss OS, not in darkmatter.

**Comment by Arunkapil01** [comment]
Yeah that's why i am requesting to fix this issue in upcoming builds.

**Comment by hmtheboy154** [comment]
It does work with Darkmatter, it just the external File Manager can't manage it and you have to use DocumentsUI to use it. 
Check if stock nougat-x86 work like Bliss, or I else I just close this as this might be nougat problem
![image](https://user-images.githubusercontent.com/39849246/127424002-ba9afdd1-db79-4329-8919-6250141bf5f3.png)
